message/message.go

```diff
@@ -32,6 +32,7 @@ const (
 	MessageTypeReceiveStatus
 	MessageTypeClose
 
+	MessageStatusFail      = uint16(0x0000)
 	MessageStatusSuccess   = uint16(0x0001)
 	MessageSatusDoReceive  = uint16(0x0002)
 	MessageSatusCheckPoint = uint16(0x0010)
```

Comment:
While you're at it, do this constants need to be exported outside of this package?